src/main/java/com/ibm/crypto/plus/provider/ECPrivateKey.java

```diff
@@ -39,7 +36,7 @@ final class ECPrivateKey extends PKCS8Key implements java.security.interfaces.EC
 
     private OpenJCEPlusProvider provider = null;
     private BigInteger s;
-    private ECParameterSpec params;
+    private transient ECParameterSpec params;
```

Comment:
Does this require the variable to be `transient` because a ECParameterSpec instance cannot be serialzed? If so is there any change in behavior here in terms of a serializable ECPrivateKey instance? I have a similar question in other classes where `transient` was added to various fields.

Reply:
Exactly. There are various other fields that are declared as `transient`. I'm not sure that we even ever serialize any of those instances. It's more of a requirement coming from the superclass. Even in the JDK code the serialization warnings are suppressed (https://github.com/ibmruntimes/openj9-openjdk-jdk17/blob/0424dfc7c8435f37d18bd32498e5e12fd2966044/src/java.base/share/classes/java/security/interfaces/ECPrivateKey.java#L50).

Reply:
Same as above. We could switch to `@SuppressWarnings(serial)`.

Reply:
I guess i dont see a behavior change here so i am ok with the code as it is.